spec/helpers/ems_cloud_helper/textual_summary_spec.rb

```diff
@@ -1,45 +1,27 @@
 require "spec_helper"
 
 describe EmsCloudHelper do
-  before do
-    controller.send(:extend, EmsCloudHelper)
-    self.class.send(:include, EmsCloudHelper)
-  end
-
-  context "textual_instances" do
+  context "#textual_instances and #textual_images" do
     def role_allows(_)
       true
     end
```

Comment:
Why is this needed?  Is it overriding something somewhere?  Can that cause test contamination?